package.json

```
@@ -17,7 +17,7 @@
     "test:coverage": "jest --coverage",
     "eslint": "eslint --ext .tsx,.js ./src/",
     "lint": "npm run eslint",
-    "format": "prettier --check --write ./src/**/*.{tsx,ts}",
+    "format": "prettier --check --write './src/app/**/*.{tsx,ts}'",
```

otherwise we would ignore `src/index.tsx`
```suggestion
    "format": "prettier --check --write './src/**/*.{tsx,ts}'",
```

---

True, fixed it!